code/_onclick/hud/rendering/plane_master.dm

```diff
@@ -113,6 +113,31 @@
 	. = ..()
 	add_filter("emissives", 1, alpha_mask_filter(render_source = EMISSIVE_RENDER_TARGET, flags = MASK_INVERSE))
 	add_filter("lighting", 3, alpha_mask_filter(render_source = O_LIGHTING_VISUAL_RENDER_TARGET, flags = MASK_INVERSE))
+
+/**
+ * Renders extremely blurred white stuff over space to give the effect of starlight lighting.
+ */
+
+/atom/movable/screen/plane_master/starlight
```

Comment:
this plane master needs to be added to a plane master controller. also, how does this work with z-mimic?

Reply:
Alright, will add it to the game plane master controller along with lighting and everything else. I have no idea how this works with z-mimic

Reply:
Wait, this doesn't need to be added to the game plane master controller because it renders onto the light plane and doesn't render to the screen.

Reply:
- You cannot see the effect on the lower layers.
- The openspace tiles are not inherently bright
- If the openspace tiles are in /area/space they will have the brightness effect

Reply:
With the correct area applied, it looks like this.
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/26465327/b36e5794-99f1-4981-955a-2c2d2b2757f5)
The black spots are from certain objects not appearing in the multi-z passthrough for some reason, as there are turfs below in those spots. This issue doesn't seem to be related though

Reply:
yea that's fine. the stuff not appearing is happening on master too